docs/release/role-handbooks/communications/README.md

```diff
@@ -0,0 +1,208 @@
+# Onboarding Notes
+
+Welcome to the Cluster API Comms Team onboarding documentation!
+
+## Overview
+
+- Understand Release Process:
+  - Get to know how project's release process works.
+  - Walk through the [release note generation process](#create-pr-for-release-notes) and try to generate notes by yourself. This is the most important process the comms team is in charge of.
```

Comment:
```suggestion
  - Walk through the [release note generation process](#create-pr-for-release-notes) and try to generate notes by yourself.
```
nit: I think that we can remove this for less clutter as the Overview should be listed by importance.